### DISCUSSION TITLE
Is ZARR an acronym?

### DISCUSSION THREAD

**Discussion by sylvaingaudan** [zarr-developers/zarr-python#1377] [General]
Hi all,
Just wondering what ZARR means. Does anyone know the origins of this name?
Thank you!

**Comment by alimanfoo** [comment]
Originally the "z" comes from the fact that it involves compression, and "arr" is short for "array".

Alternatively, it's the sound pirates make when they're sleeping :-)

**Comment by sanketverma1704** [comment]
TIL. ;)
